tests/vmi_configuration_test.go

```diff
@@ -3391,3 +3391,26 @@ func createBlockDataVolume(virtClient kubecli.KubevirtClient) (*cdiv1.DataVolume
 
 	return virtClient.CdiClient().CdiV1beta1().DataVolumes(testsuite.GetTestNamespace(nil)).Create(context.Background(), dataVolume, metav1.CreateOptions{})
 }
+
+func listCgroupThreads(pod *k8sv1.Pod) (output string, err error) {
+	virtClient := kubevirt.Client()
+
+	output, err = exec.ExecuteCommandOnPod(
+		virtClient,
+		pod,
+		"compute",
+		[]string{"cat", "/sys/fs/cgroup/cpuset/tasks"},
+	)
+
+	if err == nil {
+		// Cgroup V1
+		return
+	}
+	output, err = exec.ExecuteCommandOnPod(
```

Comment:
Can you call to `exec.ExecuteCommandOnPod` once? just extract the `[]string{...}` into variable.

Reply:
No. There is a logic between them. only if the first call fails, we call the 2nd one.

Reply:
> only if the first call fails, we call the 2nd one.

Then you probably mean to:
```
if err != nil { // If the first call fails
  exec.ExecuteCommandOnPod(...) // We call the 2nd one
}
```
But this is not how the code looks like.